docs/design/datamover.md

```diff
@@ -63,93 +63,85 @@ This design supports adding the data mover feature to the OADP operator and faci
+    resticrepository: <restic_repo>
+    sourcePVCData:
+      name: <src_pvc_name>
+      size: <src_pvc_size>
+status:
+  phase: <vsr_phase>
+  snapshotHandle: <vsc_snaphandle>
 ```
 
 We will provide a sample codebase which the vendors will be able to extend and implement their own data movers. 
 
 
 ### Default OADP Data Mover controller
 
-VolSync will be used as the default Data Mover for OADP and `restic` will be the supported method for backup & restore of PVCs. When OADP operator gets installed, VolSync will be installed alongside. Method of installation is TBD (Waiting on VolSync operator to be available. If not, we will do a  helm install). Restic repository details are configured in a `secret` object which gets used by the VolSync's resources. This design takes advantage of VolSync's two resources - `ReplicationSource` & `ReplicationDestination`. `ReplicationSource` object helps with taking a backup of the PVCs and using restic to move it to the storage specified in the restic secret. `ReplicationDestination` object takes care of restoring the backup from the restic repository. There will be a 1:1 relationship between the replication src/dest CRs and PVCs.
+VolSync will be used as the default Data Mover for OADP and `restic` will be the supported method for backup & restore of PVCs. VolSync will be installed via helm chart. Restic repository details are configured in a `secret` object which gets used by the VolSync's resources. This design takes advantage of VolSync's two resources - `ReplicationSource` & `ReplicationDestination`. `ReplicationSource` object helps with taking a backup of the PVCs and using restic to move it to the storage specified in the restic secret. `ReplicationDestination` object takes care of restoring the backup from the restic repository. There will be a 1:1 relationship between the replication src/dest CRs and PVCs.
 
-We will follow a two phased approach for implementation of this controller. For phase 1, the user will create a restic secret. Using that secret as source, the controller will create on-demand secrets for every backup/restore request. For phase 2, the user will provide the restic repo details. This may be an encryption password and BSL reference, and the controller will create restic secret using BSL info, or they can supply their own backup target repo and access credentials. We will be focussing on phase 1 approach for this design.
+We will follow a two phased approach for implementation of this controller. For phase 1, the user will create the base restic secret. Using that secret as source, the controller will create on-demand secrets for every backup/restore request. For phase 2, the user will provide the restic repo details. This may be an encryption password and BSL reference, and the controller will create restic secret using BSL info, or they can supply their own backup target repo and access credentials. We will be focussing on phase 1 approach for this design.
```

Comment:
Do we want to update this paragraph as the auto secret generation PR is merged ?

Reply:
We partly did phase 2. So I left both phases as a fallback option. WDYT?

Reply:
Maybe just add a small note about how its gonna work currently ?